setup.cfg

```diff
@@ -27,20 +27,19 @@ zip_safe = False
 packages = find:
 include_package_data = True
 install_requires =
-    dvc
-    gcsfs>=2022.7.1
+    dvc@git+https://github.com/iterative/dvc
+    gcsfs@git+https://github.com/dtrifiro/gcsfs@fix-find-dircache-corruption
```

Comment:
We install dvc from upstream in github actions. You could do the same with gcsfs. Placing github deps in here will make this package unreleasable (pip forbids this).